tests/phpunit/integration/Core/Admin/ScreensTest.php

```diff
@@ -184,4 +200,106 @@ public function test_menu_order( $given_menu_order, $expected_order ) {
 
 		$this->assertEquals( $expected_order, $menu_order, 'Menu order should match expected order for Site Kit.' );
 	}
+
+	/**
+	 * Set the initial setup setting for the current user.
+	 *
+	 * @param bool|null $complete Value for isAnalyticsSetupComplete (true/false/null).
+	 */
+	private function set_initial_setup_setting( $complete ) {
+		global $wpdb;
+		$meta_key = $wpdb->get_blog_prefix() . 'googlesitekit_initial_setup';
+		update_user_meta( get_current_user_id(), $meta_key, array( 'isAnalyticsSetupComplete' => $complete ) );
+	}
+
+	/**
+	 * Helper to load dashboard screen (simulate load- hook) capturing redirect.
+	 *
+	 * @return \Google\Site_Kit\Tests\Exception\RedirectException|null
+	 */
+	private function load_dashboard_screen() {
+		$this->screens->register();
+		do_action( 'admin_menu' );
+
+		foreach ( $this->force_get_property( $this->screens, 'screens' ) as $hook_suffix => $screen ) {
```

Comment:
We can avoid the unused `$screen` variable here:

```suggestion
		foreach ( array_keys( $this->force_get_property( $this->screens, 'screens' ) ) as $hook_suffix ) {
```